pytmc/record.py

```diff
@@ -0,0 +1,28 @@
+"""Individual EPICS Database Record creation"""
+from jinja2 import Environment, PackageLoader
+
+
+class EPICSRecord:
+    """Representation of a single EPICS Record"""
+    def __init__(self, pvname, record_type, fields=None, template=None):
+        self.pvname = pvname
+        self.record_type = record_type
+        self.fields = dict(fields) or dict()
```

Comment:
This won't work as you're expecting:
```python
In [1]: dict(None) or dict()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-149716818289> in <module>()
----> 1 dict(None) or dict()

TypeError: 'NoneType' object is not iterable
```

```suggestion
        self.fields = dict(fields) if fields is not None else {}
```